### DISCUSSION TITLE
Plugin query - spirit board, drone hacking, fossils

### DISCUSSION THREAD

**Discussion by Vemenous-Repentile** [endless-sky/endless-sky#10070] [Q&A] [answered]
Hi - apologies if this post is in the wrong place - I'm new to Github.

I recently got into the game and I've been enjoying a lot of the plugin content. 

Having started a new game and a new save file from the beginning I've enabled the plugins I wanted but I seem to be missing one which contained fossils, drone hacking and the spirit board. I was hoping someone might know which plugin(s) contain those extras. 

Thanks in advance,
VR

**Comment by ziproot** [comment]
Do you have lost in midnight?

**Reply by Vemenous-Repentile** [reply]
I do - "lost-in-midnight-main" as well as "Midnight-Expansion-main" and "Midnight-Scrapyard-main". 

I've had issues with some of the plugins not downloading with all of the content (I noticed because some image files were missing) so that might be the issue?

**Reply by ziproot** [reply, marked as answer]
I know the spirit board is from lost in midnight, so I think that is the issue.

**Reply by Vemenous-Repentile** [reply]
Thanks! Still having issues with some files not downloading in the zipped files but I've managed to get the spirit board working - much obliged
